ui-tests/src/test/java/org/openmrs/reference/CaptureVitalsTest.java

```diff
@@ -27,17 +27,12 @@
     @Before
     public void setup() {
         patient = createTestPatient();
-        createTestVisit();
-    }
-
-    @After
-    public void tearDown() throws Exception {
-        deletePatient(patient);
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+    @Test
     @Category(BuildTests.class)
-    public void captureVital() throws InterruptedException {
```

Comment:
is this a typo error, does it break when it remains captureVital

Reply:
> is this a typo error, does it break when it remains captureVital

No!. However its our role as QA Engineer to align to the OpenMRS software standards(conventions). Actually its supposed to be _captureVitalsTest()_

Reply:
Thats the exact answer i wanted from you , thanks and well done